components/ui/__tests__/EnhancedButton.test.tsx

```
@@ -0,0 +1,344 @@
+      const styles = button?.props.style;
+      
+      // Check if custom style is applied
+      const hasCustomStyle = styles?.some((style: any) => 
+        style && style.backgroundColor === 'red'
+      );
+      expect(hasCustomStyle).toBeTruthy();
+    });
+  });
+});
+
+export {};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)